measures/TableMeasures/ScalarQuantColumn.h

```diff
@@ -194,6 +194,14 @@ template<class T> class ScalarQuantColumn
   // Throw an exception if the object is null.
   void throwIfNull() const;
 
+  // get the column as a Quantum<Vector<T> >. If the units are variable, the
+  // values in the returned Vector have been converted to the
+  // unit of the 0th row entry. Otherwise, the units of the returned
+  // Quantum are itsUnit (no conversion to itsUnitOut is attempted if
+  // itsConvOut is True, because of compile issues if the column type
+  // is Complex).
```

Comment:
Quantum.tcc

This will not compile if Qtype is `Vector<Complex>`. Perhaps there is a way around it, but on the other hand, this is starting to go significantly beyond the scope of what I need this for.

`322       qVal = Qtype (qVal *
323               (qUnit.getValue().getFac()/s.getValue().getFac()));
`